<launch>

  <node pkg="line_detection" name="gabor_linedetection" type="gabor-linedetection.py" ns="left" output="screen">
    
    <param name="use_mono" type="bool" value="true" />
    <param name="use_compressed_format" type="bool" value="false" />
    <param name="subscriber_image_topic" type="str" value="image_raw" />
    <param name="publisher_image_topic" type="str" value="line_image" />
    <param name="training_file_name" type="str" value="training_for_backprojection_1.png" />
    <param name="buffer_size" type="int" value="52428800" />

    <remap from="/left/image_raw" to="/stereo_camera/left/image_raw" />
  </node>

  <node pkg="line_detection" name="gabor_linedetection" type="gabor-linedetection.py" ns="right" output="screen">
    
    <param name="use_mono" type="bool" value="true" />
    <param name="use_compressed_format" type="bool" value="false" />
    <param name="subscriber_image_topic" type="str" value="image_raw" />
    <param name="publisher_image_topic" type="str" value="line_image" />
    <param name="training_file_name" type="str" value="training_for_backprojection_1.png" />
    <param name="buffer_size" type="int" value="52428800" />

    <remap from="/right/image_raw" to="/stereo_camera/right/image_raw" />
  </node>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />
  
</launch>